helper-page.js

```diff
@@ -0,0 +1,28 @@
+exports.ToDo = class ToDo {
+  constructor(page) {
+    this.page = page;
+    this.enterTaskInput = 'input[placeholder="enter a task"]';
+    this.addButton = '//button[text()="Add"]';
+    this.doneButton = '//button[text()="done"]';
+    this.inProgressList = '//body[@class="__className_20951f"]//ul[1]';
```

Comment:
will this class change?

Reply:
For that XPath, I believe so, since it is not in our code which in most cases means that it is dynamically generated. I think we can leave it as is for now, just to get started, basically until we write some better defined selectors. Also, the last part does not work, meaning that I am unable to locate the done button corresponding to that specific task. The test passes but it does need better selectors to actually work as intended